packages/create-svelte/bin.js

```diff
@@ -100,7 +100,7 @@ const options = await p.group(
 					},
 					{
 						value: 'svelte5',
-						label: 'Try out Svelte 5 beta'
+						label: 'Try out Svelte 5 alpha'
```

Comment:
what if we did this rather than debating the semantics of alpha vs beta? is conveniently ambiguous, and articulates why you probably don't want to do this for anything serious

```suggestion
						label: 'Try the Svelte 5 preview (unstable!)'
```